lib/rules/length-zero-no-unit/index.js

```diff
@@ -16,16 +16,29 @@ const ruleMessages = require('../../utils/ruleMessages');
 const styleSearch = require('style-search');
 const validateOptions = require('../../utils/validateOptions');
 const valueParser = require('postcss-value-parser');
+//const { truncate } = require('lodash');
```

Comment:
could you please delete unneeded comments

Reply:
I have made the changes